metadata-load/src/main/java/uk/ac/ebi/ampt2d/metadata/importer/database/SraObjectsImporterThroughDatabase.java

```diff
@@ -118,6 +119,11 @@ protected Study extractAnalysisFromStudy(StudyType studyType, Study study) {
         return study;
     }
 
+    @Override
+    protected String getAccessionFromStandard(ReferenceAssemblyType.STANDARD standard) {
```

Comment:
It is counter-intuitive that a method expected to return the accession in fact doesn't. What is the reason to return always the refname when importing from the database?

Reply:
it should be for EGA case, maybe if the object has accession call super. import else, need to create RefSeq from refname.

Reply:
> It is counter-intuitive that a method expected to return the accession in fact doesn't

It is, but the reason we have to do this, again if I understand correctly, because sometimes refName is used as a _de facto_ accession in EGA, and we have to use it to identify the reference sequence. Maybe @selvaebi and @jdylan could either confirm or correct me.

Reply:
Even if EGA uses the reference name right now, that can be easily mapped to a proper accession, for instance:

* GRCh37 would have the accession GCA_000001405.1
* GRCh38 would have the accession GCA_000001405.15
* GRCh38.p13 (latest) would have the accession GCA_000001405.28

We should always favor accessions because they are guaranteed to be unique.

Reply:
Are we confined to only above three vi.z., GRCh37, GRCh38 and GRCh38.p13. 
Should we consider other reference name like `GRCh37.p13` with accession `GCA_000001405.14`

Reply:
Yes, we need to consider from GCA_000001405.1 to GCA_000001405.28.

Reply:
Mapping case-sensitive reference name to accession

- GRCh37 through GRCh37.p13 = GCA_000001405.1 through GCA_000001405.14

- GRCh38 through GRCh38.p13 = GCA_000001405.15 through GCA_000001405.28

Ref: https://www.ncbi.nlm.nih.gov/assembly/?term=GRCh37 and https://www.ncbi.nlm.nih.gov/assembly?term=GRCh38&cmd=DetailsSearch

Reply:
This has been fixed & implemented by @iyangar. Now certain human build refnames are converted to accessions